packages/ui-components/src/Input/CheckboxInput.tsx

```diff
@@ -2,10 +2,6 @@ import React from "react";
 import classNames from "classnames/bind";
 import { CommonInputProps, CommonInput } from "./CommonInput";
 
-type CheckboxProps = React.InputHTMLAttributes<HTMLInputElement>;
-
-export type CheckboxInputProps = CommonInputProps & CheckboxProps;
-
```

Comment:
`CommonInputProps` now extends `InputHTMLAttributes`, so CheckboxInput only needs to extend `CommonInputProps`